app/src/test/java/com/example/giftly/Font_SizeTests.java

```diff
@@ -0,0 +1,135 @@
+        System.out.println("Testing testOnCreate()");
+        FontSizeScreen activity = Robolectric.setupActivity(FontSizeScreen.class);
+        assertNotNull(activity);
+        System.out.println("Test passed for testOnCreate method test.");
+    }
+
+    //Test whether the value is being entered while using the font size slider
+    @Test
+    public void testFontSizeSeekBar() {
+        System.out.println("Testing testFontSizeSeekBar()");
+        FontSizeScreen activity = Robolectric.setupActivity(FontSizeScreen.class);
+        SeekBar fontSizeSeekBar = activity.findViewById(R.id.fontSizeButton);
+
+        fontSizeSeekBar.setProgress(10);
+
+        SharedPreferences preferences = activity.getSharedPreferences("prefs", Context.MODE_PRIVATE);
+        float savedSize = preferences.getFloat("font_size", 16);
+        assertEquals(14f, savedSize, 0.01f);
+        System.out.println("Test passed for testFontSizeSeekBar method test.");
+    }
+
+    //Test whether the font size will change for the xml fields: testView, editTest, and button
+    @Test
+    public void testIterateViews() {
+        System.out.println("Testing testIterateViews");
+        TextView textView = new TextView(activity.getApplicationContext());
+        EditText editText = new EditText(activity.getApplicationContext());
+        Button button = new Button(activity.getApplicationContext());
+        ViewGroup rootView = new ViewGroup(activity.getApplicationContext()) {
+            @Override
+            protected void onLayout(boolean changed, int l, int t, int r, int b) {}
+        };
+        rootView.addView(textView);
+        rootView.addView(editText);
+        rootView.addView(button);
+
+        float textSize = 24.0f;
+        activity.iterateViews(rootView, textSize);
+        assertEquals(textSize, textView.getTextSize(), 0);
+        assertEquals(textSize, editText.getTextSize(), 0);
+        assertEquals(textSize, button.getTextSize(), 0);
+        System.out.println("Test passed for testIterateViews method test.");
+    }
+
+    //Testing whether the font size actually changes with certain value
+    @Test
+    public void testSetFontSize() {
+        System.out.println("Testing testSetFontSize");
+        TextView textView = new TextView(activity.getApplicationContext());
+        ViewGroup rootView = new ViewGroup(activity.getApplicationContext()) {
+            @Override
+            protected void onLayout(boolean changed, int l, int t, int r, int b) {}
+        };
+        rootView.addView(textView);
+
+        float textSize = 14.0f;
+        activity.setFontSize(textSize);
+        assertEquals(textSize, textView.getTextSize(), 0);
+        System.out.println("Test passed for testSetFontSize method test.");
+    }
+
+    //Testing whether the save button works
+    @Test
+    public void testSaveButton() {
+        System.out.println("Testing testSaveButton()");
+        Button savingit2 = activity.findViewById(R.id.saveIt2);
```

Comment:
The variable savingit2 is not a meaningful name and may be confusing. Consider using a more descriptive variable name that indicates the purpose of the button being tested.